Backends/Wayland/package.lisp

```diff
@@ -0,0 +1,39 @@
+(in-package #:common-lisp-user)
+
+(uiop:define-package #:clim-wayland
+    (:use #:clim #:clim-lisp #:clim-backend)
+  (:local-nicknames (#:alx #:alexandria)
+                    (#:wlc #:com.andrewsoutar.cl-wayland-client)
+                    (#:wl-core #:com.andrewsoutar.cl-wayland-client/core)
+                    (#:wl-egl #:com.andrewsoutar.cl-wayland-client/egl-core)
+                    (#:xdg #:com.andrewsoutar.cl-wayland-client.protocol/stable/xdg-shell/xdg-shell))
```

Comment:
new extensions and backends should be named as `mcclim-foo`, in this case `mcclim-wayland`

Reply:
done